agent/04_agent_configure.sh

```diff
@@ -14,6 +14,7 @@ source $SCRIPTDIR/ocp_install_env.sh
 
 early_deploy_validation
 
+CLUSTER_NAMESPACE=cluster0
```

Comment:
This should be moved to environment variable instead of normal variable so it can be changed in Openshift CI.